### PR TITLE
feat:Like,comment,follow

### PR DESCRIPTION
**Pull request by shrushti2000**
Implemented like ,comment, follow feature

### CODE REVIEW

**Review by poojakelkar — APPROVED** [review]
- Nice UI
- Good Folder structure
It- Good to merge

File: src/components/PostCard/PostCard.js [hunk 1]
```diff
@@ -17,20 +22,81 @@ import {
...
 import { EditPostModal } from "../EditPostModal/EditPostModal";
-import { deleteUserPost, setPostToBeEdited } from "../../features/postSlice";
+import {
+  addComment,
+  deleteComment,
+  deleteUserPost,
+  editComment,
+  likeAndDislikePost,
+  setPostToBeEdited,
+} from "../../features/postSlice";
```
Comment (sonishreyas) [inline src/components/PostCard/PostCard.js:38]: You can explore absolute imports, it makes the code look cleaner
https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8

File: src/components/SuggestionsBar/SuggestionsBar.js [hunk 2]
```diff
@@ -8,50 +8,79 @@ import {
...
-          </Button>
-        </Flex>
-        <Flex m="10px">
-          <Avatar name="Dan Abrahmov" src="https://bit.ly/dan-abramov" />
-          <Flex flexDirection="column" m="5px">
-            <Text fontSize="xl">Adarsh Balika</Text>
-            <Text fontSize="xs">@adarshbalika</Text>
+    allUsers.length > 0 && (
+      <>
+        {" "}
```
Comment (sonishreyas) [inline src/components/SuggestionsBar/SuggestionsBar.js:41]: This can be removed

**Review by sonishreyas — COMMENTED** [review]
Code looks good. Like the folder structure

**Review by sonishreyas — APPROVED** [review]

**Pull request merged**